messages/de_DE.json

```diff
@@ -203,7 +203,7 @@
         "{contributors}\\n\\n",
         "— @YTAudioBot"
     ],
-    "donate": null,
+    "donate": "spenden",
```

Comment:
Thank you! But keep in mind that the short-named `key` is just used to access the actual string which may consist in many lines, you don't translate that; instead you have to look at the english source to see what that particular key contains and translate the `value` accordingly.

Reply:
Thanks for your feedback! However, I am not sure I understand what to change. Can you further explain what you mean by "key" and "value"? Thanks!

Reply:
Hi. Please read the [localization guidelines](https://github.com/ytaudiobot/ytaudiobot-l10n#localization-guidelines). Each whole string consists of two parts divided by a colon `:`, the `key` (left) and the` value` (right); the key is the same for every locale file and is kept untouched, what do change is the value. You don't have to translate what the `key` says, instead you have to look at the source file, which is `en_US.json`, and translate the `value`.

Reply:
Thanks! Right now I don't have the time. Hence I will close this request. But thanks for the resource!

Reply:
No problem!